datasetinsights/configs/faster_rcnn_groceries_real.yaml

```diff
@@ -44,4 +44,3 @@ optimizer:
 pretrained: False
 pretrained_backbone: True
 synchronize_metrics: True
-checkpoint_file: None
```

Comment:
I believe we need this line in all YAML files. Because if it is removed, it will get error message when we do the fine-tuning. Have you checked it?

Reply:
@adason just to check this was what you intended right? To have it removed.

Reply:
Don't we have test that validates finetuning? @BlairLee

Reply:
This has to be removed ad they don't belong to the yaml file. We should fix estimator code if it breaks something.